ui/src/main/js/page/user/UserManagement.js

```diff
@@ -4,7 +4,7 @@ import PropTypes from 'prop-types';
 import { Tab, Tabs } from 'react-bootstrap';
 import PageHeader from 'common/component/navigation/PageHeader';
 import RoleTable from 'page/user/RoleTable';
-import UserTable from 'page/user/UserTable';
+import UserTable from 'page/user/user/UserTable';
```

Comment:
seeing the directory structure of user/user/UserTable seems odd.  I understand why we have the second user directory and it makes sense to me.  
Could we change to top level user directory to usermgmt so that the directory path becomes `page/usermgmt/user/UserTable`?

In the UI the User Management is the item that is clicked to update users and roles.  So can we change the source path to reflect user management?
Then you could have the following directories for example:
```
page/usermgmt/user/UserTable
page/usermgmt/role/RoleTable
```

Reply:
I've updated the directory to reflect this comment.  It now will be set up as  
```
page/usermgmt/user/UserTable
page/usermgmt/role/RoleTable
```